src/tests/functional/plugin/shared/src/snippets/mha.cpp

```diff
@@ -81,9 +81,16 @@ void MHA::generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticSha
     for (int i = 0; i < model_inputs.size(); ++i) {
         const auto& model_input = model_inputs[i];
         ov::Tensor tensor;
+        ov::test::utils::InputGenerateData in_data;
+<<<<<<< HEAD
         // To avoid big relative errors in the vicinity of zero, only positive values are generated for bf16 precision
-        int start_from = model_input.get_element_type() == ov::element::bf16 ? 0 : -1;
-        tensor = ov::test::utils::create_and_fill_tensor(model_input.get_element_type(), model_input.get_shape(), 2, start_from, 256);
+        in_data.start_from = model_input.get_element_type() == ov::element::bf16 ? 0 : -1;
+=======
+        in_data.start_from = -1;
+>>>>>>> d89271e8c1 (Apply comments)
```

Comment:
Please do not overwrite the [recent fix on input generation for bf16 MHA test cases](https://github.com/openvinotoolkit/openvino/pull/21250).

Reply:
Thanks for the comment.
This is probably rebase issue. I manually kept your fix. But I don't see it now.

Reply:
Updated. Your fix is kept